test/acceptance/Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG SIMPLEVERSION_VERSION
 COPY . /dist
 # Force copying of native lib into root of tool - don't pollute LD_LIBRARY_PATH
 RUN dotnet tool install SimpleVersion.Tool --version ${SIMPLEVERSION_VERSION} -g --add-source /dist \
-    && cp /root/.dotnet/tools/.store/simpleversion.tool/${SIMPLEVERSION_VERSION}/simpleversion.tool/${SIMPLEVERSION_VERSION}/tools/netcoreapp3.1/any/runtimes/alpine.3.9-x64/native/* /root/.dotnet/tools/.store/simpleversion.tool/${SIMPLEVERSION_VERSION}/simpleversion.tool/${SIMPLEVERSION_VERSION}/tools/netcoreapp3.1/any
+    && cp /root/.dotnet/tools/.store/simpleversion.tool/${SIMPLEVERSION_VERSION}/simpleversion.tool/${SIMPLEVERSION_VERSION}/tools/netcoreapp3.1/any/runtimes/linux-x64/native/* /root/.dotnet/tools/.store/simpleversion.tool/${SIMPLEVERSION_VERSION}/simpleversion.tool/${SIMPLEVERSION_VERSION}/tools/netcoreapp3.1/any
```

Comment:
@GlatBane  Is there a reason this has switched from the alpine specific native version?

All acceptance tests are now failing - so something has broken here

Reply:
@Kieranties When using the new preview LibGit2Sharp package the resulting NuGet package does no longer have OS specific binaries compared with using the latest stable release (0.26.2 from 2019), only Architecture specific for (Linux/OSX/Windows).

I don't know if this actually needs a fix from the LibGit2Sharp. or just make it incompatible with this repo (I have only tested it on Ubuntu with the linux-x64). or if the versions for the specific Linux flavors are only generated for the stable releases.

Reply:
I've dug into this a little further today and have been able to identify that for `3.1-bionic`, `3.1-focal` and `3.1-buster` the change is all good - in fact we do not need to copy any native libraries any where at all.

For `3.1-alpine` however, the issue exists.  I've attempted to validation against the `3.14` version of the image but am getting pull errors from MCR.

If you change the target tag to `3.1-focal` and remove the force copy of native libraries this can be unblocked.

I'll address the alpine issue in another PR

Reply:
I have updated the tag and removed the copy of the native library, also had to change the way git was installed as focal don't have apk but apt as package management system

Reply:
Thanks for iterating on this - I'll get this merged in now, but I have some additional tests to add in to the build pipeline.  Once those are there I'll get things released.